c++/src/capnp/BUILD.bazel

```diff
@@ -1,5 +1,9 @@
 load("@capnp-cpp//src/capnp:cc_capnp_library.bzl", "cc_capnp_library")
 
+package(default_visibility = [
+    "@//:__subpackages__",
+])
+
 cc_library(
     name = "capnp",
     srcs = [
```

Comment:
All project targets for Cap'n'Proto itself are now only visible within this codebase. This should prevent users from depending directly on targets within `//src/...`.